### ISSUE TITLE
i18n for UI?

### ISSUE THREAD

**Issue by PaRaD1SE98** [enricoros/nextjs-chatgpt-app#89]
As title

**Comment by enricoros** [comment]
@PaRaD1SE98 : good idea. This could actually be automated ;) What would be your target language? I'm looking for translators :D

**Comment by PaRaD1SE98** [comment]
I can do Chinese and Japanese. Just want to have an arch to work on.

**Comment by enricoros** [comment]
@PaRaD1SE98 have you used next-intl before? I am thinking of using that library, and if you're still game for the translations, I'll go through the process of extracting all strings..

**Comment by NilsJacobsen** [comment]
Hi folks, 

Did you have a look at [ParaglideJS](https://inlang.com/m/gerre34r/library-inlang-paraglideJs). Might be the best-performing i18n library right now and it's easy - no async handling. -> [nextjs guide](https://inlang.com/g/wxcebbig/guide-lorissigrist-useParaglideJsWithNextjsAppRouter)
You also get a no-code [editor](https://inlang.com/m/tdozzpar/app-inlang-finkLocalizationEditor) and [ide extension](https://inlang.com/m/r7kp499g/app-inlang-ideExtension) ootb.

**Comment by PeterDaveHello** [comment]
I may be able to help with Traditional Chinese.

**Comment by enricoros** [comment]
Folding #377 in here. It was an empty [i18n] stub from a second user, so it adds no requirements - only one more distinct person asking for localization, which is the gate I named: demand decides when the framework gets prioritized. This ticket stays the canonical one for the i18n framework and the ES/IT/CN/JA locales.

**Comment by enricoros** [comment]
Decision: i18n happens, via a standard framework rather than ad-hoc string tables - next-intl is the current candidate. No date; the first concrete step is string extraction, and this thread gets pinged when that starts, translators first. ES/IT/CN/JA volunteers: the offers from 2023 are remembered and appreciated.
